docs/instances.rst

```diff
@@ -467,6 +468,108 @@ and ``set2``, which contains ``instance2`` and ``instance3``.
 Instance sets will also be useful when using the :ref:`command line interface <CommandLineReference>` of
 simexpal and when defining the :ref:`RunMatrix`.
 
+.. _PostprocessInstances:
+
+Postprocessing
+--------------
+
+There might be cases where you need to process the instances after installing or downloading them, before they
+are ready to be used in the experiments. In order to do so, you can use the
+
+- ``postprocess``: list or string of postprocessing arguments
+
+key. Afterwards, you can install and postprocess the instances by calling
+
+.. code-block:: bash
+
+   $ simex instances install
+
+in the terminal.
+
+Before processing an instance, simexpal copies the contents of each file belonging to an instance into separate
+``<filename>.original`` files. After postprocessing an instance simexpal creates an ``<instance_name>.postprocessed``
+file, signalling the successful postprocessing of an instance. If an error occurs during the postprocessing of an
+instance, the original instance files will be restored and the postprocessing will be skipped.
+
+Arbitrary Postprocessing
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can define arbitrary postprocessing steps by setting the ``postprocess`` key to a list of dictionaries
+containing the
+
+- ``args``: list of postprocessing arguments
+- ``environ``: dictionary of (environment variable, value)-pairs
+- ``workdir``: path of the working directory
+
+keys.
+
+Assume you want to postprocess the ``facebook_combined`` and ``cit-HepTh`` network from
+`SNAP <https://snap.stanford.edu/data/>`_ using two executables ``postprocess1`` and ``postprocess2``, which
+take the path of the instance as parameter. Also, you have to prepend the path for ``postprocess1`` to the
```

Comment:
This is just to demonstrate the possibility/necessity of adding the appropriate PATH-variable in simexpal in order to find the executable for the postprocessing. There might be cases where the PATH was added beforehand, so that is isn't necessary to do so anymore.